cypress/support/Train.js

```diff
@@ -8,15 +8,18 @@ import * as trainDialogsGrid from './components/TrainDialogsGrid'
 import * as editDialogModal from './components/EditDialogModal'
 import * as helpers from './Helpers'
 
-function Today() { return Cypress.moment().format("MM/DD/YYYY") }
+let currentTrainingSummary
+let originalTrainingSummary
+let isBranched
+let originalChatMessages
+let editedChatMessages
```

Comment:
Better than using `window` but I think this still has the same affect of creating a global.

Reply:
I've done a lot of reading about let and globals and it seems that this is not creating a true global. It appears that the distinctions are subtle, but that it does not have the same pitfalls as the global can have.

However, we can discuss how you would like this sort of thing to be done when we meet up on Wednesday.